services/app-web/src/pages/StateSubmission/StateSubmissionForm.tsx

```diff
@@ -333,6 +341,7 @@ export const StateSubmissionForm = (): React.ReactElement => {
                         <Contacts
                             draftSubmission={formDataFromLatestRevision}
                             updateDraft={updateDraftHealthPlanPackage}
+                            previousDocuments={previousDocuments}
                         />
```

Comment:
I think we are passing `previousDocuments` here to satisfy `HealthPlanFormPageProps` type in the `Contacts` component. I'm wondering the reason for passing a prop that is never used vs specifying the prop types in `Contacts`, like 
```
export const Contacts = ({
    draftSubmission,
    showValidations = false,
    updateDraft,
}: {
    draftSubmission: HealthPlanFormPageProps['draftSubmission']
    showValidations?: boolean,
    updateDraft: HealthPlanFormPageProps['updateDraft'],
}): React.ReactElement => {
```

Reply:
I can do that! No specific reason why its written the way it is.